recipes/filetype/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "filetype" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "fe3afe00d40b7102b78ffd7ab85defad13657a976ebbcb818b2fdfbba3888c06" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - examples
+    - filetype
+    - filetype.types
+
+about:
+  home: https://github.com/h2non/filetype.py
+  license: MIT
+  license_family: MIT
+  license_file: ''
```

Comment:
You can add here `LICENSE`

Reply:
The license file is not distributed in the pypi compressed file. Is it better to use the github source instead in this case?

Reply:
@druvus -- please see https://conda-forge.org/docs/meta.html#packaging-the-license-manually for how to manually add the license to the conda package.

Reply:
thanks @mwcraig! License file is now manually added.